### ISSUE TITLE
EE6 - Flyout menu does not accommodate long list of channels

### ISSUE THREAD

**Issue by inoatech** [ExpressionEngine/ExpressionEngine#572]
Second-level flyout menu does not accommodate long list of channels. It only shows the last few, depending on screen height. Would be nice if it had a scroll bar when needed.

**Comment by ops-andy** [comment]
@inoatech can you provide a screenshot please so we know where to replicate this. 
thanks

**Comment by inoatech** [comment]
Thanks Andy. Here's one.

<img width="638" alt="ScreenShot" src="https://user-images.githubusercontent.com/245898/96762560-09cba180-139f-11eb-99c0-e752fa3eec54.png">

**Comment by intoeetive** [comment]
Solution suggested in https://github.com/ExpressionEngine/ExpressionEngine/issues/718 (does add a potentially ugly scrollbar however)

```
.dropdown{
    ...
    max-height: 100vh;
    overflow-y: scroll;
}
```

**Comment by ops-andy** [comment]
fixed in 6.0.1
